### PR TITLE
chore: Improve Code Style

### PR DESCRIPTION
**Pull request by yezz123**
@aminalaee I like the Idea behind SQLAdmin, I will add some features to it in the future and I will use it in one of my projects soon, While I read the code I found some parts that could be refactored and enhanced to be better.

If you have any questions, we could discuss them!

### CODE REVIEW

**Comment by aminalaee** [comment]
Hey @yezz123 ,
Glad you liked it and will use it.

Any feedback and contribution would be appreciated :)

**Comment by aminalaee** [comment]
I think the codecov precision needs to be configured. I can do it in a separate PR, but if you're interested we can fix it here.

**Comment by yezz123** [comment]
> I think the codecov precision needs to be configured. I can do it in a separate PR, but if you're interested we can fix it here.

Yes That's would be great!

**Comment by aminalaee** [comment]
Looks like I can't edit your fork,

can you try the following codecov configuration in `.codecov.yml`?

```
coverage:
  precision: 2

comment: false
```

**Comment by yezz123** [comment]
I just add it, I don't know why GitHub didn't show the button when we give access to write 🤔

File: .codecov.yml [hunk 1]
```diff
@@ -1 +1,4 @@
-comment: off
+coverage:
+  precision: 1
+
+comment: false
```
Comment (aminalaee) [inline .codecov.yml:4]:
```suggestion
coverage:
  status:
    project:
      default:
        target: auto
        threshold: 1%

comment: false

```

**Review by aminalaee — APPROVED** [review]
Thanks for this.

**Pull request merged**